sys-fs/udiskie/udiskie-1.1.1.ebuild

```diff
@@ -1,54 +0,0 @@
-# Copyright 1999-2018 Gentoo Foundation
```

Comment:
This is the last stable version, so you can't remove it until the new version is tested and goes stable.